app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,43 @@
+// TODO
+// 1. Read Collection - GET /tasks => 완료
+// 2. Read Item - GET /tasks/{id}
+// 3. Create - POST /tasks => 완료
+// 4. Update - PUT/PATCH /tasks/{id}
+// 5. Delete - DELETE /tasks/{id}
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.repositories.TaskRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private TaskRepository taskRepository;
+
+    public TaskController(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
```

Comment:
여기에 `final`을 추가해 줍시다.
```suggestion
    private final TaskRepository taskRepository;

    public TaskController(TaskRepository taskRepository) {
        this.taskRepository = taskRepository;
    }
```

Reply:
@johngrib 질문이 있습니다, 상수를 선언 할 때는 static final로 선언해야 한다는 점을 공부를 했는데요, TaskRepository의 객체의 경우 값이 변화될 수 있는데 왜 `final` 키워드를 사용해야 할까요?

Reply:
값의 개념에 대해 혼동하고 계신 것 같아요. 여기에서 `final`로 고정한 것은 `TaskRepository`의 레퍼런스입니다. `TaskRepository`가 관리하는 값이 아니에요. `TaskRepository`를 다른 `TaskRepository`로 교체하지 않고 계속 사용하겠다는 거죠.